src/zoid/buttons/component.jsx

```diff
@@ -877,6 +881,17 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
           return props?.displayOnly || [];
         },
       },
+
+      message: {
+        type: "object",
+        queryParam: true,
+        required: false,
+        validate: ({ props, value = {} }) => {
+          // $FlowFixMe
+          validateButtonMessage(props, value);
+        },
+        default: () => ({}),
```

Comment:
Will a default value of `{}` always turn on messages? We want this to be an opt-in feature so if so we likely want to remove that.

Reply:
good call, will remove